icons/react.svg

```diff
@@ -1 +1 @@
\ No newline at end of file
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>React icon</title><path d="M7.268 1.325c-.47-.015-.897.083-1.268.297-1.44.83-1.766 3.146-.94 6.375C1.845 8.9 0 10.34 0 11.997s1.846 3.098 5.06 4c-.826 3.226-.5 5.543.94 6.373a2.38 2.38 0 0 0 1.205.306c1.333 0 3.026-.95 4.795-2.673 1.77 1.722 3.46 2.673 4.795 2.673A2.38 2.38 0 0 0 18 22.37c1.44-.83 1.766-3.147.943-6.373 3.21-.902 5.057-2.342 5.057-4 0-1.66-1.846-3.1-5.057-4.002.824-3.227.497-5.543-.943-6.373-1.413-.814-3.64.08-6 2.375-1.742-1.692-3.41-2.632-4.732-2.672zm-.063.99c1.058 0 2.536.862 4.096 2.38A24.847 24.847 0 0 0 9.25 7.242a25.122 25.122 0 0 0-3.232.5c-.685-2.687-.508-4.678.486-5.25.2-.116.435-.175.703-.175zm9.402.01c.344-.032.644.02.89.164.995.572 1.173 2.563.487 5.25-1.006-.227-2.09-.395-3.232-.5h-.002a24.85 24.85 0 0 0-2.05-2.544c1.465-1.428 2.877-2.277 3.907-2.37zM12 5.42a23.414 23.414 0 0 1 1.445 1.73 29.785 29.785 0 0 0-2.892 0A23.552 23.552 0 0 1 12 5.42zm-.066 2.696a29.03 29.03 0 0 1 2.263.082 28.585 28.585 0 0 1 2.198 3.8 29.006 29.006 0 0 1-2.198 3.796c-1.48.11-2.914.11-4.394 0a29.114 29.114 0 0 1-2.198-3.797 28.477 28.477 0 0 1 2.198-3.8 29.03 29.03 0 0 1 2.13-.08zm3.546.205a23.48 23.48 0 0 1 2.227.388 23.42 23.42 0 0 1-.78 2.115A30.006 30.006 0 0 0 15.48 8.32zm-6.96.003c-.262.408-.517.82-.756 1.232a30.3 30.3 0 0 0-.692 1.268 23.53 23.53 0 0 1-.78-2.115 23.657 23.657 0 0 1 2.23-.385zm-3.204.627a24.752 24.752 0 0 0 1.182 3.047 24.95 24.95 0 0 0-1.182 3.045c-2.674-.752-4.314-1.9-4.314-3.045 0-1.146 1.64-2.296 4.314-3.047zm13.368 0c2.674.752 4.314 1.9 4.314 3.047 0 1.145-1.64 2.293-4.314 3.045-.305-.986-.702-2.005-1.182-3.043l.002-.003h-.002a24.788 24.788 0 0 0 0-.002 24.788 24.788 0 0 0 1.182-3.045zM12 9.862a2.14 2.14 0 1 0 0 4.277 2.14 2.14 0 1 0 0-4.278zM7.072 13.17a30.354 30.354 0 0 0 .692 1.266 30.388 30.388 0 0 0 .74 1.21c-.77-.094-1.508-.217-2.207-.374a23.307 23.307 0 0 1 .775-2.103zm9.856 0a23.357 23.357 0 0 1 .78 2.116 23.63 23.63 0 0 1-2.226.383 29.6 29.6 0 0 0 1.446-2.5zM6.016 16.25a24.976 24.976 0 0 0 3.23.5 24.976 24.976 0 0 0 .002 0v.003a24.998 24.998 0 0 0 .002 0 24.998 24.998 0 0 0 2.05 2.543c-1.956 1.903-3.81 2.772-4.798 2.21-.994-.574-1.172-2.567-.486-5.255zm11.968 0c.686 2.69.508 4.68-.486 5.254-.2.115-.436.172-.703.172-1.058 0-2.536-.862-4.096-2.38a24.674 24.674 0 0 0 2.048-2.543h.004l.002-.002a25.02 25.02 0 0 0 3.23-.5zm-7.433.588c.49.024.973.04 1.45.04.476 0 .96-.014 1.447-.038a23.53 23.53 0 0 1-1.45 1.735 23.596 23.596 0 0 1-1.446-1.737z"/></svg>
```

Comment:
SVG source found [here](https://github.com/facebook/create-react-app/blob/282c03f9525fdf8061ffa1ec50dce89296d916bd/test/fixtures/relative-paths/src/logo.svg) in the `create-react-app` package.

Comparing PR path (red) to SVG (black) there appear to be some differences.
![image](https://user-images.githubusercontent.com/1830867/102872695-fccb3d00-4437-11eb-8e66-9d65d5185d88.png)

Feel free to try the path I generated from source:
```svg
M24 12.002c0-1.59-1.991-3.097-5.044-4.031.704-3.111.391-5.587-.988-6.379a2.149 2.149 0 00-1.091-.274v-.005c-1.345 0-3.106.959-4.887 2.622-1.78-1.653-3.542-2.602-4.887-2.602v.005c-.411 0-.783.088-1.106.274-1.374.792-1.682 3.263-.973 6.364C1.981 8.916 0 10.417 0 12.002c0 1.59 1.991 3.097 5.044 4.031-.704 3.111-.391 5.587.988 6.379.318.186.69.274 1.101.274 1.345 0 3.106-.959 4.887-2.622 1.781 1.653 3.542 2.603 4.887 2.603.411 0 .783-.088 1.106-.274 1.375-.792 1.683-3.263.974-6.364C22.019 15.094 24 13.587 24 12.002zm-7.128-9.597v.005c.225 0 .406.044.558.127.665.382.954 1.834.729 3.703-.054.46-.142.944-.249 1.438a23.628 23.628 0 00-3.106-.533 23.768 23.768 0 00-2.035-2.446c1.591-1.48 3.086-2.292 4.103-2.294zM7.666 14.507c.23.391.465.773.704 1.145a22.352 22.352 0 01-2.006-.386c.181-.631.406-1.282.66-1.932.201.39.412.782.642 1.173zM6.35 8.74a23.381 23.381 0 012.015-.386 27.392 27.392 0 00-1.34 2.332A21.07 21.07 0 016.35 8.74zm1.247 3.277a26.293 26.293 0 012.192-3.821 25.683 25.683 0 014.412-.004 25.641 25.641 0 012.202 3.806 26.293 26.293 0 01-2.192 3.821 25.303 25.303 0 01-4.412.005 25.489 25.489 0 01-2.202-3.807zm8.737 2.475c.225-.391.435-.788.636-1.179.264.656.489 1.311.675 1.947-.641.157-1.316.289-2.015.391.24-.376.479-.762.704-1.159zm-.005-4.994a27.53 27.53 0 00-.704-1.145c.695.103 1.365.23 2.006.386a22.082 22.082 0 01-.66 1.932 23.433 23.433 0 00-.642-1.173zm-2.984-2.461c-.44-.02-.89-.034-1.345-.034-.46 0-.915.01-1.36.034.44-.572.895-1.096 1.345-1.565.455.469.91.993 1.36 1.565zM6.541 2.556c.147-.088.338-.127.563-.127v-.005c1.013 0 2.514.807 4.109 2.28a23 23 0 00-2.02 2.441 22.715 22.715 0 00-3.112.538 14.701 14.701 0 01-.255-1.419c-.23-1.869.054-3.321.715-3.708zM5.308 14.977a15.047 15.047 0 01-1.37-.499c-1.732-.739-2.852-1.707-2.852-2.475s1.12-1.742 2.852-2.475c.421-.181.881-.342 1.355-.494a23.48 23.48 0 001.101 2.979 23.142 23.142 0 00-1.086 2.964zM7.123 21.6a1.173 1.173 0 01-.553-.132c-.665-.382-.954-1.834-.729-3.703.054-.46.142-.944.249-1.438.959.235 2.006.416 3.106.533.66.905 1.345 1.727 2.035 2.446C9.637 20.788 8.14 21.6 7.123 21.6zm4.887-3.067c-.455-.47-.91-.993-1.36-1.565.44.02.89.034 1.345.034.46 0 .915-.01 1.36-.034-.44.572-.895 1.096-1.345 1.565zm5.449 2.916c-.147.088-.338.127-.563.127-1.013 0-2.514-.807-4.109-2.28a23 23 0 002.02-2.441 22.726 22.726 0 003.111-.538c.113.494.201.969.254 1.424.232 1.868-.052 3.321-.713 3.708zm2.598-6.971c-.421.181-.881.342-1.355.494a23.48 23.48 0 00-1.101-2.979c.45-1.018.812-2.011 1.086-2.965.484.152.944.318 1.375.499 1.732.739 2.852 1.707 2.852 2.475-.005.768-1.125 1.742-2.857 2.476zM9.759 12.002a2.236 2.236 0 114.472 0 2.236 2.236 0 01-4.472 0z
```

Reply:
Good catch @adamrusted!